src/routes/search/search-vendors.js

```diff
@@ -73,6 +73,28 @@ export default class SearchVendors extends Component {
     return !searchVendors || searchVendors.isPending;
   }
 
+  renderButtonGroup() {
+    const { location, history } = this.props;
+    const goTo = (url) => () => history.push(url);
+    const isActive = (url) => url === location.pathname;
+
+    const searchButtons = [
+      ['Vendors', '/eholdings/vendors'],
+      ['Packages', '/eholdings/packages'],
+      ['Titles', '/eholdings/titles']
+    ];
+
+    return (
+      <div style={{ float: 'left', marginRight: '1rem' }}>
```

Comment:
Yea, I didn't want to add a `search.css` file in the routes and also didn't want to bother breaking out another component just to float the buttons next to the form. 😅